libs/http-parser/Makefile

```diff
@@ -7,30 +7,27 @@
 
 include $(TOPDIR)/rules.mk
 
-PKG_NAME:=libhttp-parser
-PKG_VERSION:=2.3.0
+PKG_NAME:=http-parser
+PKG_VERSION:=2.7.1
 PKG_RELEASE=1
 PKG_MAINTAINER:=Ramanathan Sivagurunathan <ramzthecoder@gmail.com>
 PKG_LICENSE:=MIT
 PKG_LICENSE_FILES:=LICENSE-MIT
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
-PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
-PKG_SOURCE_URL:=git://github.com/joyent/http-parser.git
-PKG_SOURCE_PROTO:=git
-PKG_SOURCE_VERSION:=56f7ad0e2e5a80f79d214015c91e1f17d11d109f
-
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
```

Comment:
It's fine as it is since you're pulling a tarball. It's preferred over using git.